docs/csharp/language-reference/keywords/init.md

```diff
@@ -0,0 +1,36 @@
+---
+description: "init keyword - C# Reference"
+title: "init keyword - C# Reference"
+ms.date: 03/10/2017
+f1_keywords: 
+  - "init"
```

Comment:
@davidwengier Do we need this? or only init_CSharpKeyword is sufficient?

Reply:
Certainly don't need it from Roslyn's point of view.

Reply:
@BillWagner The articles in the _keywords_ folder have two entries under `f1_keywords` (sometimes in reverse order):

```
f1_keywords: 
  - "<keyword>_CSharpKeyword"
  - "<keyword>"
``` 

Is the second entry superfluous?